mne/preprocessing/maxwell.py

```diff
@@ -544,20 +478,31 @@ def _update_info(raw, origin, int_order, ext_order, nsens, nmoments):
         raw file object with raw.info modified
     """
     from .. import __version__
-    # TODO: Flesh out/fix bookkeeping info
+    # TODO: Continue to fill out bookkeeping info as additional features
+    # are added (fine calibration, cross-talk calibration, etc.)
+    int_moments = get_num_moments(int_order, 0)
+    ext_moments = get_num_moments(0, ext_order)
+
+    # Get time in sec, usec
+    now = time.time()
+    date_now = np.array([int(np.floor(now)),
+                         int(1e6 * (now - np.floor(now)))])
```

Comment:
DRY this away with your other implementation please, maybe in `mne.io.write` have
```
def _date_now()
    now = time.time()
    return np.array([int(...), int(...)])
```